c-dependencies/js-compute-runtime/xqd.h

```diff
@@ -108,49 +151,52 @@ typedef enum BodyWriteEnd {
 
 // Module fastly_abi
 WASM_IMPORT("fastly_abi", "init")
-int xqd_init(uint64_t abi_version);
+FastlyStatus xqd_init(uint64_t abi_version);
```

Comment:
I think it's fine to leave all the host call definitions as returning `int` values, we can build a more typed api on top of them.

Reply:
I've reverted them back to returning `int` values and have a new function which converts `int` to `FastlyStatus` and asserts the conversion was possible, I.E. will crash if trying to convert an integer which is not included as an enumeration in FastlyStatus